2.4/test/run

```diff
@@ -342,10 +342,121 @@ noprealloc = true" > $config_file
     echo "  Success!"
 }
 
+function insert_and_wait_for_replication() {
+    local host
+    host=$1
+    local data
+    data=$2
+
+    # Storing document into replset and wait replication to finish
+    CONTAINER_IP="${host}" ADMIN_PASS="adminPassword" \
+    mongo_admin_cmd "db.getSiblingDB('test_db').data.insert(${data});
+        while(true) {
+          var status=rs.status();
+          var optime=status.members[0].optime;
+          var ok=true;
+          for(var i=1; i < status.members.length; i++) {
```

Comment:
Yes and no:-)

First member value is stored `optime` variable. And values of every other member are compared to this first member value.